src/views/viewer/ChannelView/ChannelView.style.tsx

```diff
@@ -77,11 +47,9 @@ export const Title = styled(Text)`
 
 export const SubTitle = styled(Text)`
   ${fluidRange({ prop: 'fontSize', fromSize: '14px', toSize: '18px' })};
-
   padding: ${sizes(1)} ${sizes(2)};
-  margin-top: ${sizes(2)};
-  color: ${colors.white};
-  background-color: ${colors.gray[800]};
+  margin-top: ${sizes(1)};
```

Comment:
```suggestion

  margin-top: ${sizes(1)};
```
Stylelint says: ` Expected empty line before declaration (declaration-empty-line-before)`